internal/controllers/servergroup/tests/servergroup-create-full/00-servergroup.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
Can you rename this file to `00-create-resource.yaml` like I did in https://github.com/k-orc/openstack-resource-controller/pull/417/, to bring more consistency across tests?